test/src/kbasesearchengine/test/search/ElasticIndexingStorageTest.java

```diff
@@ -1000,4 +1004,45 @@ public void testLookupInKey() throws Exception{
         assertThat("overlapping ranges did not return intersection", hits5.guids, is(set(guid2)));
     }
 
+
+    @Test
+    public void addHighlighting() throws Exception {
+        GUID guid1 = new GUID("WS:11/1/2");
+        GUID guid2 = new GUID("WS:11/2/2");
+        GUID guid3 = new GUID("WS:11/3/2");
+        try {
+            prepareTestMultiwordSearch(guid1, guid2, guid3);
+        } catch(Exception e) {
+            //TODO: remove items from each test after completion. Will error if run all sequence in order due to version conflict
```

Comment:
The elastic indexes are cleared after every test

Reply:
after each test in the the test file? I thought it runs the prep functions and then just runs the tests in sequence

Reply:
Yep, see the method marked with the @After annotation